Maturity levels

robmoffat opened the pull request.

psmulovics commented:
@jstclair2019 you really have to clear that CLA :D

jstclair2019 commented:
> @jstclair2019 you really have to clear that CLA :D
@psmulovics argh, yeah I know! 😄

robmoffat commented:
Hi All,

Check out the maturity levels I've added, starting here:

https://deploy-preview-103--open-source-readiness.netlify.app/docs/bok/OSMM/Introduction

I'm aiming to reconcile both TODO OSPO maturity and OSPO Alliance levels.  Also, I've added indexing so you should be able to see which activities correspond to which levels (this will be important as we complete the work on the developer maturity model in the SIG)

thanks!

psmulovics commented on docs/bok/Activities/Security-Testing.md:
```diff
@@ -0,0 +1,45 @@
+  - OSPO (Role)
+  - Developer (Role)
+  - Security Expert (Role)
+  - Placeholder
```
Is this a placeholder entry? Seems pretty well written for that.

jstclair2019 replied:
@psmulovics I think so. We want to allow for additional roles as we define them (security expert could delineate into multiple roles in CISO, vulnerability management, etc)

psmulovics replied:
That is creating a tag called 'Placeholder' and adds this document to it :) So it's not a placeholder for the role.

robmoffat replied:
Hi yes - I've tagged all the documents "Placeholder" where we need to add some decent content.

For this one, I think there are several roles Jim came up with (CISO, TISO etc) that have might roll under the category of "Security Expert". What do you think?

robmoffat commented:
@jstclair2019 if you go onto your LFX profile and associate with the Linux Foundation you might be able to get the green light for the EasyCLA process - should be something you have to accept.

jstclair2019 commented:
> @jstclair2019 if you go onto your LFX profile and associate with the Linux Foundation you might be able to get the green light for the EasyCLA process - should be something you have to accept.

LFX walked me through the same contributor process that you did, so it should be sitting in @maoo or someone's queue to approve..?

maoo commented:
/easycla

psmulovics commented on docs/bok/OSMM/Introduction.md:
```diff
@@ -8,17 +8,17 @@ tags:
 - The model can also be used as a tool for benchmarking an organization's open source maturity against that of other organizations in the same industry or sector. 
 - The OSMM can provide guidance to organizations on how to improve their open source practices and to align these with their overall business objectives. 
 - The ultimate goal of an OSMM is to enable organizations to maximize the benefits of open source software while managing the associated risks and costs effectively.
+- Since there are lots of [Activities](../Activities/Introduction) in this body of knowledge, the maturity levels provide some guidance about the _order_ to tackle activities.  Organisastions beginning their open source journey are advised to start with activities categorized as [Level 1](Level-1) and proceed from there.
```
typo: Organisastions

robmoffat replied:
ta
